src/main/java/org/opentripplanner/routing/trippattern/TripTimes.java

```diff
@@ -226,6 +230,26 @@ public void setServiceCode(int serviceCode) {
         }
     }
 
+    /**
+     * @return 2D array for via names for each stop in sequence.
+     */
+    private String[][] makeViasArray(final Collection<StopTime> stopTimes) {
+        String[][] vias = new String[stopTimes.size()][];
+
+        int i = 0;
+        for (final StopTime st : stopTimes) {
+            if (st.getVia() == null) {
+                vias[i] = new String[0];
```

Comment:
Maybe use a constant for the `new String[0]` here and below.

Reply:
I've added a constant for empty string array.